utils/common.js

```diff
@@ -0,0 +1,19 @@
+export function isEmpty(text) {
+	return !text;
+}
+export function isDuplicated(text){
+	var duplicated=false;
+	$('#tourneyPlayersList td').each(function(){
+		var playernameinthisRow=$(this).text().toLowerCase();
+		if(text.toLowerCase()===playernameinthisRow){
+			duplicated=true;
+		}
+	});
+	return duplicated;
```

Comment:
There's an Array.find method in ES6 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find